torch/_inductor/cudagraph_trees.py

```diff
@@ -385,7 +385,11 @@ def deferred_cudagraphify(inputs: list[InputType]) -> OutputType:
         copy_misaligned_inputs(inputs, check_input_idxs)
 
         fn, out = cudagraphify(model, inputs, new_static_input_idxs, *args, **kwargs)
-        fn = align_inputs_from_check_idxs(fn, inputs_to_check=check_input_idxs)
+        # cudagraph will already clones input locally, no need to copy back
+        mutated_input_idxs = OrderedSet()
```

Comment:
no need for OrderedSet here if you add default arg. you also don't need to add additional mypy annotations :)

Reply:
I meant a default arg of None. Reading the code in `align_inputs_from_check_idxs` passing in None and an empty `OrderedSet` seems equivalent since we never add to the set and guard on None

```
ret_pair_defined = return_pair_idxs is not None
```